neuralmonkey/decoders/sequence_decoder.py

```diff
@@ -0,0 +1,310 @@
+"""Abstract class for autoregressive decoding.
+
+Either for the recurrent decoder, or for the transformer decoder.
+
+The sequence decoder uses the while loop to get the outputs. Descendants should
```

Comment:
* už je z toho _autogressive_ decode místo sequence.
* ~~the~~ _a_ while loop
* while loop jede, dokude nevygeneruje `</s>`  symbol nebo nedosáhne maximální délky